app/charts/bar/bars-stacked-state.tsx

```diff
@@ -415,79 +416,80 @@ const useBarsStackedState = (
   const bounds = useChartBounds(width, margins, height);
   const { chartWidth, chartHeight } = bounds;
 
-  xScale.range([0, chartWidth]);
-  xScaleInteraction.range([0, chartWidth]);
-  xScaleTimeRange.range([0, chartWidth]);
-  yScale.range([chartHeight, 0]);
+  yScale.range([0, chartWidth]);
+  yScaleInteraction.range([0, chartWidth]);
+  yScaleTimeRange.range([0, chartWidth]);
+  xScale.range([chartHeight, 0]);
 
   const isMobile = useIsMobile();
 
   // Tooltips
   const getAnnotationInfo = useCallback(
-    (datum: Observation): TooltipInfo => {
-      const bw = xScale.bandwidth();
-      const x = getX(datum);
+    (datum: Observation): TooltipInfoInverted => {
+      const bw = yScale.bandwidth();
+      const y = getY(datum);
 
-      const tooltipValues = chartDataGroupedByX.get(x) as Observation[];
-      const yValues = tooltipValues.map(getY);
+      const tooltipValues = chartDataGroupedByY.get(y) as Observation[];
+      const xValues = tooltipValues.map(getX);
       const sortedTooltipValues = sortByIndex({
         data: tooltipValues,
         order: segments,
         getCategory: getSegment,
         sortingOrder: "asc",
       });
-      const yValueFormatter = getStackedTooltipValueFormatter({
+      const xValueFormatter = getStackedTooltipValueFormatterInverted({
         normalize,
-        yMeasureId: yMeasure.id,
-        yMeasureUnit: yMeasure.unit,
+        xMeasureId: xMeasure.id,
+        xMeasureUnit: xMeasure.unit,
         formatters,
         formatNumber,
       });
```

Comment:
Nit: Would it still work to have `getStackedTooltipValueFormatter` but with `measureId` and `measureUnit` properties? I think it should still be clear, as we can only have one measure in bar chart or column chart, so technically it shouldn't matter if it's x or y measure, because there can only be one anyways 😅